src/ap/hostapd.c

```diff
@@ -176,8 +176,10 @@ static void hostapd_clear_old(struct hostapd_iface *iface)
 	 * allow them to use the BSS anymore.
 	 */
 	for (j = 0; j < iface->num_bss; j++) {
+#ifndef CONFIG_SONIC_HOSTAPD
```

Comment:
The design mentions that the hostapd receive packets directly when the mode is configured as MAB. Can you please help with the APIs for that functionality?

Reply:
Sorry, I didn't get the question. The hostapd doesn't or never communicates with MAB directly. It communicates only with PACD.  Similarly MAB communicates only with PACD. Can you please point to the section in the design document, if it is mentioned.. will correct the same.

Reply:
sorry my bad. Somehow I remember seeing that & wanted to check